Resources/Prototypes/SS220/Entities/Clothing/OuterClothing/SRTCoat.yml

```
@@ -0,0 +1,16 @@
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: ClothingOuterSRTCoat
+  name: плащ ГСН
+  description: Бронекостюм тёмно-синего цвета. Отличная защита от большинства видов урона.
```

```suggestion
  description: Бронекостюм тёмно-синего цвета. Отличная защита от большинства видов урона.
```
всм бронекостюм? статы те же, что и у парадного плаща выше

---

там плащ как плащ, в плане одежда, а не в плане плащ, как у обычного главы.